superset/viz.py

```diff
@@ -2347,8 +2351,14 @@ def get_properties(self, d):
         d[self.deck_viz_key] = path
         if line_type != 'geohash':
             del d[line_column]
+        d['metric'] = d.get(self.metric_label)
```

Comment:
Does that do anything? I thought the `query_obj` only took `metrics` as an arg (?)

Reply:
It's leftover, will remove.